src/components/brokers-down/brokers-down-slide.tsx

```diff
@@ -41,28 +41,32 @@ const BrokersDownLinkText = styled.a`
   color: #0068ea !important;
   text-decoration: none;
   ${BrokersDownLinkSpan}:hover & {
-  text-decoration: none;
-  color: #0068ea !important;
-  }`;
+    text-decoration: none;
+    color: #0068ea !important;
+  }
```

Comment:
move this directly to the styles under `BrokersDownLinkSpan` (line 37).

it'll look like this:
```
BrokersDownLinkSpan = styled.span`
  &:hover {
    ...
  }
`
```

Reply:
Strangely, this did not work.